src/scss/mixins/_radio-checkbox.scss

```diff
@@ -179,63 +185,65 @@
 /// @param {string} $class - The block level class name (BEM naming convention).
 /// @output Styles for radios which look like a group of buttons, where one can be selected.
 @mixin oFormsRadioButtonsStyled($class: 'o-forms') {
-    @include oNormaliseVisuallyHidden;
-    $sidePadding: 32px;
-    & + .#{$class}__label {
-        @include oColorsFor(o-forms-radio-buttons-styled, text);
-        cursor: pointer;
-        font-size: $_o-forms-field-default-font-size;
-        font-weight: oFontsWeight('semibold');
-        padding: 5px $sidePadding 7px;
-        position: relative;
-        text-align: center;
-        transition: 0.3s background-color, 0.15s color ease-out;
-        display: inline-block;
-        border-left: 1px solid oColorsGetColorFor(o-forms-radio-buttons-styled, border);
-        &:first-of-type {
-            margin-left: 3px;
-            border-left: 0;
-        }
-        &:before {
-            content: '';
-            position: absolute;
-            top: -3px;
-            right: -3px;
-            bottom: -3px;
-            left: -3px;
-            border: 1px solid oColorsGetColorFor(o-forms-radio-buttons-styled, border);
-            border-left: 0;
-            border-right: 0;
+    @include oBrandConfigureFor('o-forms') {
+        @include oNormaliseVisuallyHidden;
+        $sidePadding: 32px;
+        & + .#{$class}__label {
+            color: oBrandGet('radio-buttons-styled-text');
+            cursor: pointer;
+            font-size: $_o-forms-field-default-font-size;
```

Comment:
Could we wrap two in one with this (and the line below) and use `oTypographySansBold(x)` or something?

Reply:
As they're semibold, I don't think so.

Reply:
In terms of creating something like `oTypographySansSemiBold(x)` or `oTypographySansBold(x, $kinda-bold: true)`, It probably makes sense to highlight this as a custom outlier rather than make `o-typography` more permissive.

Reply:
I think `o-typography` only really uses Metric semibold 🤔 
I'm slightly confused now. 

https://github.com/Financial-Times/o-typography/blob/71f5d6576971c5c8085832a289db51c4d1f6402e/src/scss/_mixins.scss#L9

https://github.com/Financial-Times/o-typography/blob/71f5d6576971c5c8085832a289db51c4d1f6402e/src/scss/_mixins.scss#L149